src/Controls/src/Core/WeakEventManager.cs

```diff
@@ -87,7 +87,7 @@ public void RemoveEventHandler<TEventArgs>(EventHandler<TEventArgs> handler, [Ca
 			RemoveEventHandler(eventName, handler.Target, handler.GetMethodInfo());
 		}
 
-		public void RemoveEventHandler(EventHandler handler, [CallerMemberName] string eventName = null)
+		public void RemoveEventHandler(Delegate? handler, [CallerMemberName] string eventName = "")
```

Comment:
I know this is already merged, but may I ask, what's the point of making it nullable, if you thrown if it is null?

Reply:
@dersa It’s to support `INotifyPropertyChanged`

Here’s the information I included on it from the PR:

> ### WeakEventManager.cs
> - Enables nullable (`#nullable enable`)
>   - Required to support `INotifyPropertyChanged`
>     - [`INotifyPropertyChanged.PropertyChanged` event is nullable](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.inotifypropertychanged.propertychanged?view=net-6.0&WT.mc_id=mobile-43581-bramin) as of .NET 5.0
>       ```cs
>       event System.ComponentModel.PropertyChangedEventHandler? PropertyChanged;
>       ```
>     - Screenshot of the null reference error if nullable is not enabled on WeakEventManager:
>         <img width="1024" alt="image" src="https://user-images.githubusercontent.com/13558917/140237613-26d0c60c-a5c6-4939-813a-6c7e6a033ba7.png">
>